pySDC/projects/DAE/misc/ProblemDAE.py

```diff
@@ -54,14 +54,30 @@ def solve_system(self, impl_sys, u0, t):
         me : dtype_u
             Numerical solution.
         """
-
         me = self.dtype_u(self.init)
-        opt = root(
-            impl_sys,
-            u0,
-            method='hybr',
-            tol=self.newton_tol,
-        )
-        me[:] = opt.x
+
+        def implSysAsNumpy(unknowns, **kwargs):
+            me.diff[:] = unknowns[: np.size(me.diff)].reshape(me.diff.shape)
+            me.alg[:] = unknowns[np.size(me.diff) :].reshape(me.alg.shape)
+            sys = impl_sys(me, **kwargs)
+            return np.append(sys.diff.flatten(), sys.alg.flatten())  # TODO: more efficient way?
+
+        if type(me) == DAEMesh:
```

Comment:
We don't need this anymore, right? We should be able to plug this into scipy solvers directly.